src/tools/dev/scripts/bv_support/bv_hdf4.sh

```diff
@@ -16,7 +16,11 @@ function bv_hdf4_disable
 
 function bv_hdf4_depends_on
 {
-    echo "szip zlib"
+    if [[ "$DO_ZLIB" == "yes" ]] ; then
+        echo "szip zlib"
+    else
+        echo "szip"
```

Comment:
So, HDF4 depends UNCONDITIONALLY on szip?

Reply:
HDF4 previously unconditionally depended upon both szip and zlib being built by build_visit.  I simply removed zlib from being required to be built by build_visit. If zlib not requested, system zlib is attempted.